src/views/home/index.tsx

```diff
@@ -21,90 +22,41 @@ const Home = () => {
-            <p className="font-geist text-dev-black-300 font-body2-regular dark:text-dev-purple-300">
-              Explore our GitHub repository for comprehensive documentation, code examples, and contributions. Join our developer community to enhance your Polkadot projects.
-            </p>
-          </PDLink>
-          <PDLink
-            to="/code?s=1"
-            className={cn(
-              'group',
-              'relative',
-              'flex flex-1 flex-col',
-              'p-6',
-              'min-h-[320px]',
-              '!text-current',
-              'border border-transparent hover:border-dev-pink-500',
-              'transition-border-colors duration-200 ease-linear',
-              'bg-dev-purple-100 dark:bg-dev-black-900',
-
-              'after:pointer-events-none after:absolute after:opacity-0 after:content-[""]',
-              'after:-right-3/4 after:-top-3/4 after:size-3/4 after:bg-dev-purple-500 after:blur-[150px]',
-              'after:transition-opacity after:duration-200',
-              'hover:after:opacity-100',
-              'overflow-hidden',
-            )}
-          >
-            <Icon
-              name="icon-linkArrow"
-              size={[24]}
+        <div className="grid w-full grid-cols-1 gap-4 md:grid-cols-3">
+          {HOME_LINKS.map((card) => (
+            <PDLink
+              key={card.to}
+              to={card.to}
               className={cn(
-                'absolute right-4 top-4',
-                'transition-transform duration-200 ease-linear',
-                'group-hover:-translate-y-1 group-hover:translate-x-1',
+                'group relative flex flex-1 flex-col p-6 !text-current',
+                'border border-transparent hover:border-dev-pink-500',
+                'transition-all duration-200 ease-linear',
+                'bg-dev-purple-100 dark:bg-dev-black-900',
+                'overflow-hidden',
+                'after:pointer-events-none after:absolute after:opacity-0 after:content-[""]',
+                'after:-right-3/4 after:-top-3/4 after:size-3/4 after:bg-dev-purple-500 after:blur-[150px]',
+                'after:transition-opacity after:duration-200',
+                'hover:after:opacity-100',
               )}
-            />
-            <Icon
-              name="icon-brackets"
-              className="mb-8 text-dev-pink-500"
-              size={[40]}
-            />
-            <h4 className="mb-2 font-h4-bold">
-              Developer Console
-            </h4>
-            <p className="font-geist text-dev-black-300 font-body2-regular dark:text-dev-purple-300">
-              Dive into our advanced developer console for seamless on-chain interaction and rapid prototyping. Experience a streamlined, user-friendly environment designed to accelerate your development process and boost productivity.
-            </p>
-          </PDLink>
+            >
+              <Icon
+                name="icon-linkArrow"
+                size={[24]}
```

Comment:
24 is the default value of <Icon /> component from a while (we can check the hole project and make a clean up)